### DISCUSSION TITLE
How to use css from node modules without global importing?

### DISCUSSION THREAD

**Discussion by anuj-choudhary** [vercel/next.js#13666] [Help] [answered]
I am using ```React-Slick``` in my next.js app. When I am importing `slick.css` and `slick-theme.css` globally then it's working fine. But I want to have modular imports to reduce page loading time. But in that case, it's not loading perfectly. 

### Screen Shots

1. With global importing
![Screenshot 2020-06-02 at 7 19 46 PM](https://user-images.githubusercontent.com/61403508/83528828-358cde80-a507-11ea-8a7a-06a0f904b2c7.png)

2. With modular importing
![Screenshot 2020-06-02 at 7 20 17 PM](https://user-images.githubusercontent.com/61403508/83528877-40477380-a507-11ea-80f0-fe71b67524c4.png)

### System Information
OS: macOS

`Note:- I have overridden a few classes of react-slick`

**Comment by jamesmosier** [comment, marked as answer]
Currently there is no reliable way to do this. There is an open issue related to this: https://github.com/vercel/next.js/issues/12079
